example/client/main.go

```diff
@@ -3,32 +3,23 @@ package main
 import (
 	"bytes"
 	"flag"
+	"fmt"
 	"io"
 	"net/http"
 	"sync"
 
 	quic "github.com/lucas-clemente/quic-go"
 	"github.com/lucas-clemente/quic-go/h2quic"
-	"github.com/lucas-clemente/quic-go/internal/protocol"
-	"github.com/lucas-clemente/quic-go/internal/utils"
 )
 
 func main() {
-	verbose := flag.Bool("v", false, "verbose")
 	tls := flag.Bool("tls", false, "activate support for IETF QUIC (work in progress)")
 	flag.Parse()
 	urls := flag.Args()
 
-	if *verbose {
-		utils.SetLogLevel(utils.LogLevelDebug)
-	} else {
-		utils.SetLogLevel(utils.LogLevelInfo)
-	}
-	utils.SetLogTimeFormat("")
```

Comment:
While I agree with disentangling the example from the internal packages in general, I'm using -v very much when developing, so at this point, I'm not a fan of merging anything that makes my life harder.

Reply:
Isn't it equivalent to just setting the env var `QUIC_GO_LOG_LEVEL=DEBUG`?

Reply:
No, it's not, because the time format will be different. I don't think it's too hard to remove this code when copy-pasting the example, so I'd prefer not to make any change to the example client and server at this point.

Reply:
We could introduce another ENV for the format if you need that.

I think making the example useful for people is quite important, and having to edit the code makes the barrier of entry much harder.

Alternatively, we could split it into several examples, one set of "simple" ones that work right away, and more involved ones we can use for debugging. WDYT?

Reply:
Then you'd replace `go run example/main.go -v` by `QUIC_GO_LOG_LEVEL=debug DONT_SHOW_TIMESTAMPS=true go run example/main.go` to have a decent output. I don't think this provides any usability benefit to new users.

Furthermore, I don't think this is hard. We had a lot of people play with quic-go, and open all kinds of trivial issues, but this never caused any problems.
I'm don't want to merge any changes that make further development of quic-go harder for me.

Reply:
Okay, then I vote for splitting up the examples into a simple client & server and a debug client & server with more features.

Reply:
We can do that in a separate PR though.

Reply:
That sounds like a good idea.